### PR TITLE
Extend docs on how to use Minikube

### PR DESCRIPTION
**Pull request by chaudum**
## Summary of the changes / Why this is an improvement

Without the correct storage class, the tests would run into timeouts.

## Checklist

 - [ ] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: docs/source/development.rst [hunk 1]
```diff
@@ -87,23 +87,54 @@ passed via the ``--kube-config`` and ``--kube-context`` arguments to pytest_.
...
 
-.. note::
+Minikube
+........
 
-   If you are using macOS in combination with Minikube, you need to set up a
-   `Minikube tunnel`_ in order to get an external IP address assigned to the
-   CrateDB services.
+Using minikube_ as Kubernetes backend for the tests requires a few
+preparations. Make sure, your minikube has configured enough resources, for
```
Comment (SStorm) [inline docs/source/development.rst:94]: The commas are not necessary here?

```suggestion
preparations. Make sure your minikube has configured enough resources for
```

**Review by SStorm — APPROVED** [review]
One minor comment, but looks good!

File: docs/source/development.rst (hunk 1 above)
Comment (chaudum) [reply docs/source/development.rst:94]: No, don't think so :+1:

File: docs/source/development.rst [hunk 2]
```diff
@@ -87,23 +87,54 @@ passed via the ``--kube-config`` and ``--kube-context`` arguments to pytest_.
...
 
-.. note::
+Minikube
+^^^^^^^^
 
-   If you are using macOS in combination with Minikube, you need to set up a
-   `Minikube tunnel`_ in order to get an external IP address assigned to the
-   CrateDB services.
+Using minikube_ as Kubernetes backend for the tests requires a few
+preparations. Make sure your minikube has configured enough resources for
```
Comment (matthijskrul) [inline docs/source/development.rst:94]: A little more natural is "For example, make sure your Minikube...". Also think about whether you want to capitalize Minikube or not - either is fine but should be consistent

Comment (chaudum) [reply docs/source/development.rst:94]: I think lower case is correct. https://minikube.sigs.k8s.io/docs/

Comment (chaudum) [reply docs/source/development.rst:94]: If it's lower case, do you mean it should also be lower case in the title?

**Pull request merged**